propel/provider.go

```diff
@@ -30,13 +30,27 @@ func Provider() *schema.Provider {
 				DefaultFunc: schema.EnvDefaultFunc("PROPEL_CLIENT_SECRET", nil),
 				Description: "Your Propel Application's secret.",
 			},
+			"oauth_url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   false,
+				DefaultFunc: schema.EnvDefaultFunc("PROPEL_OAUTH_URL", nil),
+				Description: "The Propel OAuth URL",
+			},
+			"api_url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   false,
+				DefaultFunc: schema.EnvDefaultFunc("PROPEL_API_URL", nil),
+				Description: "The Propel API URL",
+			},
```

Comment:
It would be useful to have these already 🤔

Reply:
I know ;)